### PR TITLE
fix: linked form known issue - recovery tutorial 

### PR DESCRIPTION
**Pull request by nathansandi**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by megglos — CHANGES_REQUESTED** [review]
Thank you ❤️ Let's also link this sub-section from the related announcement see https://github.com/camunda/camunda-docs/pull/3328

Other than that some suggestions, let me know what you think

**Review by megglos — APPROVED** [review]
LGTM! One more thing, we also need to put this into the next branch of the docs, can you update this @nathansandi please? Then we are ready for DevEx review :)

**Comment by megglos** [comment]
@christinaausley we need this published asap for a running incident, can you please take a look and release after merge with prio? 🥺 Thank you 😀

File: docs/components/modeler/web-modeler/advanced-modeling/form-linking.md [hunk 1]
```diff
@@ -48,6 +52,35 @@ To deploy to a Camunda 8 cluster with a version lower than 8.4, linked forms wil
 This conversion will only be applied to the XML deployed to the cluster; the diagram in Web Modeler will not be changed.
 :::
 
+#### Known issues with linked forms
+
+##### Wrong form used for user task - [zeebe/#16311](https://github.com/camunda/zeebe/issues/16311)
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/advanced-modeling/form-linking.md:57]: H4 headers and beyond make the text quite small, so launching an issue in my backlog to address this across our docs.

File: docs/components/modeler/web-modeler/advanced-modeling/form-linking.md [hunk 2]
```diff
@@ -48,6 +52,35 @@ To deploy to a Camunda 8 cluster with a version lower than 8.4, linked forms wil
...
+
+Some users have encountered an issue that was present in Camunda `8.4.0`, `8.4.1`, and `8.4.2` where linked forms did not correspond correctly to their tasks, leading to discrepancies in workflow execution. We have identified and rectified this issue in the Camunda `8.4.3` release, ensuring that tasks are now generated with the correct forms, as intended in your workflow design.
+
+###### How to fix the issue - Regenerate tasks with correct forms
+
+To correct any instances affected by this issue, we recommend the following steps after updating to Camunda `8.4.3`:
+
+1. Navigate to [Operate](/components/operate/operate-introduction.md) to access your workflow instances impacted by the linked form issue.
+2. Locate the instances where tasks were generated with the incorrect forms.
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/advanced-modeling/form-linking.md:66]: Should the user then click on an instance? Click on the process instance key? How do I get to the screen with the wrench icon as mentioned in the next step?

File: docs/components/modeler/web-modeler/advanced-modeling/form-linking.md [hunk 3]
```diff
@@ -48,6 +52,35 @@ To deploy to a Camunda 8 cluster with a version lower than 8.4, linked forms wil
...
+
+<img src={IssueLinkedFormSolution01} style={{width: 400}} alt="Modify Process Instance on Operate" />
+
+4. After this, a popup explaining how process modification works will appear. Click **Continue**, select the active task, and click **Cancel instance**.
+
+<img src={IssueLinkedFormSolution02} style={{width: 400}} alt="process instance modification mode" />
+
+<img src={IssueLinkedFormSolution03} style={{width: 400}} alt="cancel instance" />
+
+5. Add a flow node.
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/advanced-modeling/form-linking.md:77]: Would add details on how to do this. What should I click?

File: docs/components/modeler/web-modeler/advanced-modeling/form-linking.md [hunk 4]
```diff
@@ -48,6 +52,35 @@ To deploy to a Camunda 8 cluster with a version lower than 8.4, linked forms wil
...
+
+1. Navigate to [Operate](/components/operate/operate-introduction.md) to access your workflow instances impacted by the linked form issue.
+2. Locate the instances where tasks were generated with the incorrect forms.
+3. Move the instance to the same task, effectively restarting the task. To do this, click the wrench icon in the top right corner:
+
+<img src={IssueLinkedFormSolution01} style={{width: 400}} alt="Modify Process Instance on Operate" />
+
+4. After this, a popup explaining how process modification works will appear. Click **Continue**, select the active task, and click **Cancel instance**.
+
+<img src={IssueLinkedFormSolution02} style={{width: 400}} alt="process instance modification mode" />
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/advanced-modeling/form-linking.md:73]: I moved these images around based on the referenced steps and adjusted the alt text. Does this look correct?

**Review by christinaausley — COMMENTED** [review]
A few comments to clear up and make this accessible -- then we can merge and release 👍

File: docs/components/modeler/web-modeler/advanced-modeling/form-linking.md [hunk 5]
```diff
@@ -48,6 +52,35 @@ To deploy to a Camunda 8 cluster with a version lower than 8.4, linked forms wil
...
+
+###### How to fix the issue - Regenerate tasks with correct forms
+
+To correct any instances affected by this issue, we recommend the following steps after updating to Camunda `8.4.3`:
+
+1. Navigate to [Operate](/components/operate/operate-introduction.md) to access your workflow instances impacted by the linked form issue.
+2. Locate the instances where tasks were generated with the incorrect forms.
+3. Move the instance to the same task, effectively restarting the task. To do this, click the wrench icon in the top right corner:
+
+<img src={IssueLinkedFormSolution01} style={{width: 400}} alt="Modify Process Instance on Operate" />
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/advanced-modeling/form-linking.md:69]: I don't think this is blocking, but is there a reason why the images aren't referenced as ?

File: docs/components/modeler/web-modeler/advanced-modeling/form-linking.md (hunk 2 above)
Comment (berkaycanbc) [reply docs/components/modeler/web-modeler/advanced-modeling/form-linking.md]: @christinaausley Updated.

File: docs/components/modeler/web-modeler/advanced-modeling/form-linking.md (hunk 3 above)
Comment (berkaycanbc) [reply docs/components/modeler/web-modeler/advanced-modeling/form-linking.md]: @christinaausley updated.

File: docs/components/modeler/web-modeler/advanced-modeling/form-linking.md (hunk 5 above)
Comment (nathansandi) [reply docs/components/modeler/web-modeler/advanced-modeling/form-linking.md:69]: there is an import on the beggining of the file :) 

import IssueLinkedFormSolution01 from './img/linked_issue01.png';
import IssueLinkedFormSolution02 from './img/linked_issue02.png';
import IssueLinkedFormSolution03 from './img/linked_issue03.png';
import IssueLinkedFormSolution04 from './img/linked_issue04.png';

File: docs/components/modeler/web-modeler/advanced-modeling/form-linking.md (hunk 4 above)
Comment (nathansandi) [reply docs/components/modeler/web-modeler/advanced-modeling/form-linking.md:73]: Sorry, It was not clear for me this @christinaausley

**Review by christinaausley — APPROVED** [review]

**Pull request merged**